Chapter2_ProgramStructure/Chapter 2 #1.JS

```diff
@@ -0,0 +1,14 @@
+function f1()
```

Comment:
Style guide: 

function f1() {	
    var t = "";
    for(var i = 1; i < 8; i++) {
        for(var j=1;j<=i;j++) {
            t += "*";
        }

      	console.log(t);

      	t = "";
    }
}

f1();